api/admin/controller.py

```diff
@@ -1692,6 +1692,10 @@ def _get_integration_protocols(cls, provider_apis, protocol_name_attr="__module_
             if cardinality != None:
                 protocol['cardinality'] = cardinality
 
+            supports_registration = getattr(api, "SUPPORTS_REGISTRATION", None)
+            if supports_registration != None:
+                protocol['supports_registration'] = supports_registration
+
```

Comment:
Is this currently used or is solely it forward-looking?

Reply:
It's used in https://github.com/NYPL-Simplified/circulation-web/pull/144, to determine whether to display the library registration UI for a collection or service.